index.d.ts

```diff
@@ -58,4 +56,4 @@ declare module 'pushwoosh-react-native-plugin' {
 
   declare const Pushwoosh: Pushwoosh;
   export = Pushwoosh;
-}
```

Comment:
this was automatically done by the github web editor. if it is a problem, let me know and I'll try and revert it